app/models/record.rb

```diff
@@ -657,6 +661,49 @@ def get_section(section_name)
     end
   end
 
+  # Should return a list where each entry is a (title,[title_list])
+  def mmt_sections
+    section_list = []
+
+    path = File.join(Rails.root,"/data/ummc_section_titles.json")
+    json = JSON.load(File.read(path))
+    field_list = []
+    map = {}
+    json.each do |obj|
+      display_name = obj["displayName"]
+      field_list << display_name
+      map[display_name] = obj["properties"]
+    end
+
+    field_list.each do |title|
+      section = self.get_mmt_section(title, map)
+      if section[0][1].count > 0
+        section_list += section
+      end
+    end
+
+    used_titles = (section_list.map {|section| section[1]}).flatten
+    all_titles = self.sorted_record_datas.map { |data| data.column_name }
+
+    others = [["Other", all_titles.select {|title| !used_titles.include? title }]]
```

Comment:
What is Other for?

Reply:
It is other fields not included in the MMT section lists.  It is also a catch all if we ever add fields that are not included in the section lists, they will show up here